src/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+}
+
+&__image {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+&__title {
+  line-height: 18px;
+  font-size: 12px;
+  font-weight: 500;
+  color: $main-text-color;
+  margin-bottom: 16px;
+}
+
+&__product-code {
+  @include other-text;
+
+  margin-bottom: 16px;
+}
+
+&__reviews {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+&__reviews-count {
+  @include other-text;
+
+  text-align: right;
+}
+
+&__price {
+  display: flex;
+  align-items: center;
+  margin-bottom: 16px;
+  justify-content: space-between;
+}
+
+&__price-text {
+  line-height: 18px;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+&__price-cost {
+  line-height: 18px;
+  font-size: 16px;
+  font-weight: 700;
+  color: $main-text-color;
+}
+
+&__button {
+  display: flex;
+  box-sizing: border-box;
+  background-color: $accent-color;
+  text-decoration: none;
+  color: #fff;
+  line-height: 16px;
+  font-weight: 700;
+  font-size: 14px;
+  text-transform: uppercase;
+  align-items: center;
+  justify-content: center;
+  transition: all 0.2s ease;
+  height: 40px;
+
+  border-radius: $border-radius;
+  border: 1px solid $accent-color;
```

Comment:
The variable `$accent-color` is used here. Confirm its definition in your SCSS files to avoid compilation issues.